docker/node/bin/entrypoint.sh

```diff
@@ -4,6 +4,13 @@ set -e
 uid=$(stat -c %u /srv)
 gid=$(stat -c %g /srv)
 
+# Caddy certificates handling
+# This allows internal HTTP requests to the Caddy container
+if [ -f /srv/build/caddy/pki/authorities/local/root.crt ] && [ ! -f /usr/local/share/ca-certificates/caddy.crt ]; then
+    cp /srv/build/caddy/pki/authorities/local/root.crt /usr/local/share/ca-certificates/caddy.crt
+    update-ca-certificates
+fi
```

Comment:
Should be executed only once when starting the container: adds the root crt, and update certs in the node-debian container